requirements.txt

```diff
@@ -1,65 +1,156 @@
+aiohttp==3.8.1
```

Comment:
@matentzn I'm having an issue with Python dependencies. Basically, the best practice is to lock in specific versions of all of the resolved transitive dependencies, and install those for consistency.

I have created an issue for this: #43

For now, I've implemented a temporary solution which I describe further in that issue.

Reply:
Ok!